alembic/env.py

```diff
@@ -29,7 +29,8 @@
 def get_pg_url() -> str:
     url = (
         f"postgres+psycopg2://{os.getenv('POSTGRESQL_USER')}"
-        f":{os.getenv('POSTGRESQL_PASSWORD')}@postgres:5432/{os.getenv('POSTGRESQL_DATABASE')}"
+        f":{os.getenv('POSTGRESQL_PASSWORD')}@{os.getenv('POSTGRESQL_HOST', 'postgres')}"
+        f":5432/{os.getenv('POSTGRESQL_DATABASE')}"
```

Comment:
How about making the port also configurable, with a default of "5432"?

Reply:
sure :D

Reply:
actually, how about we imported this method from packit_service :O